<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resetpassword</class>
 <widget class="QWidget" name="resetpassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: rgb(219, 215, 255);
background:rgb(255, 255, 255);
font-size:px;
	font: 16pt &quot;楷体&quot;;
font-style:MingLiU-ExtB;
}
QFrame{
border-style:outset;
border-radius:15px;
border:sold 10px rgba(0,0,0);
background-image:url(:/pic/picture/loginBackground.jpg);
}
QLineEdit{
color:#8d98a1;
background-color:#405361;
font-size:24px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/pic/picture/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="userName">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passWord">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>OldPassword</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passWord_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>NewPassword</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dbName">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background-color:#405361;
color:white;
border-radius:10px;
border-style:outset;
font-style:MingLiU-ExtB;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
